New OK dokument version 1.2 implemented

SoliteaSlovensko opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

SoliteaSlovensko committed:
OK document files via upload

SoliteaSlovensko committed:
Updated swagger

SoliteaSlovensko committed:
Fix JSON format

SoliteaSlovensko committed:
Fix validation errors from build

SoliteaSlovensko committed:
connector category fix

SoliteaSlovensko committed:
solved issues found in review

SoliteaSlovensko committed:
Merge branch 'microsoft:dev' into dev

SoliteaSlovensko committed:
Merge branch 'microsoft:dev' into dev

SoliteaSlovensko committed:
Merge branch 'microsoft:dev' into dev

SoliteaSlovensko committed:
Extended feature signature request

Added new paramater for signature request with option to disable feature "Sign Anywhere"
Fixed response result 200 from Download document action.

SoliteaSlovensko committed:
Merge branch 'microsoft:dev' into dev

SoliteaSlovensko committed:
Add files via upload

New version of OK dokument API implemented. Currect version 1.2

SoliteaSlovensko committed:
Fixed JSON format

SoliteaSlovensko committed:
Fixed JSON Format

Amjed-Ayoub commented on independent-publisher-connectors/OK dokument/SECURITY.md:
```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.7 BLOCK -->
```
@SoliteaSlovensko this file is not required in the open source repository. you can remove it.

Amjed-Ayoub commented on independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:
```diff
@@ -157,7 +157,8 @@
                   "x-ms-visibility": "important",
                   "enum": [
                     "tosign",
-                    "preprocessing_tags"
```
@SoliteaSlovensko An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. can you please confirm that this update will not break existing users' flow and apps?

SoliteaSlovensko replied:
Removing this enum will not have impact on existing flows or apps, values defined in emuns are still valid and string send to request will work.

vmanoharas commented:
Hello @SoliteaSlovensko, 

Please provide update on Amjed comment below.

SoliteaSlovensko committed:
Merge branch 'microsoft:dev' into dev

SoliteaSlovensko committed:
Update apiDefinition.swagger.json

New API version
Added new field manupulation into signature request
Modified Additional rules according to documentation v 1.3

SoliteaSlovensko committed:
Update README.md

extended description how to work with signature request rules
